modules/manage/pages/kubernetes/troubleshooting/troubleshoot.adoc

```diff
@@ -20,10 +20,86 @@ To check the values that were set in the Redpanda Helm chart:
 
 [,bash]
 ----
-helm get values redpanda -n redpanda --all
+helm get values redpanda --namespace <namespace> --all
 ----
 
-== HelmRelease is not ready
+== View Redpanda logs
+
+Logs are sent directory to STDOUT for each Pod. Use the command below to view the logs for a Pod:
+
+```bash
+kubectl logs <pod-name> --namespace <namespace>
+```
+
+To change the log level to `debug`:
+
+[tabs]
+======
+Helm + Operator::
++
+--
+.`redpanda-cluster.yaml`
+[,yaml]
+----
+apiVersion: cluster.redpanda.com/v1alpha1
+kind: Redpanda
+metadata:
+  name: redpanda
+spec:
+  chartRef: {}
+  clusterSpec:
+    logging:
+      logLevel: debug
+----
+
+```bash
+kubectl apply -f redpanda-cluster.yaml --namespace <namespace>
+```
+--
+
+Helm::
++
+--
+[tabs]
+====
+--values::
++
+.`logging.yaml`
+[,yaml]
+----
+logging:
+  logLevel: debug
+----
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --values logging.yaml --reuse-values
+```
+
+--set::
++
+```bash
+helm upgrade --install redpanda redpanda/redpanda --namespace <namespace> --create-namespace \
+  --set logging.logLevel=debug
+```
+====
+--
+======
+
+== View Redpanda Operator logs
```

Comment:
Can we add additional segment where to understand what happen using operator and Redpanda resource would be to

```
kubectl get events --namespace <namespace> --sort-by='.metadata.creationTimestamp'

kubectl get helmreleases.helm.toolkit.fluxcd.io -o yaml <REDPANDA_RESOURCE_NAME>

kubectl get redpandas.cluster.redpanda.com -o yaml 
```

In both helmreleases and redpandas the condition shows what is going on with our helm installation. The events should have duplicated information that conditions have in both resources.